src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        this.color = color;
+        this.side = side;
```

Comment:
The constructor should not contain expressions. It's better to create a separate variable for the 'side' and then pass it to the constructor for better readability.